spec/fd_spec.rb

```diff
@@ -187,6 +187,39 @@ class Bar; def self.table_name() :bars end end
     cm.name.must_equal "CM"
   end
 
+  if defined?(Account) && defined?(Address)
```

Comment:
I believe this should be on top of this [line](https://github.com/jeremyevans/fixture_dependencies/pull/32/files#diff-afca79d3167ae6dc914cad682490734cf057c26b88a32072d9576c19fd82b941R223) otherwise it will be skipped